packages/api-docs/src/generatePageJsonFiles.ts

```diff
@@ -1,9 +1,12 @@
+import * as fs from 'fs';
 import * as path from 'path';
 import * as fse from 'fs-extra';
 import { ApiModel } from '@microsoft/api-extractor-model';
 import { IPageJsonOptions } from './types';
 import { generatePageJson } from './pageJson';
 
+const tmpFile = path.join(__dirname, 'tmp.json');
```

Comment:
nit, maybe give this a more descriptive name (also ideally it ought to go under `os.tmpdir()` to be very sure it doesn't accidentally get checked in)